### PR TITLE
done with toggling

### PR DESCRIPTION
**Pull request by MugaboDenys**

### CODE REVIEW

File: src/components/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React from "react"
+
+
```
Comment (mutesialine) [inline src/components/Navbar.js]: Also here

File: src/components/Main.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from "react"
+
```
Comment (mutesialine) [inline src/components/Main.js]: Remove this import from react. Since you're using the new version of react. We no longer need it.

**Review by mutesialine — CHANGES_REQUESTED** [review]
Great work @MugaboDenys; I just left a few comments below. To make your code much clear.

File: src/components/Main.js (hunk 2 above)
Comment (MugaboDenys) [reply src/components/Main.js]: Yeah sure!
Thanks

**Review by mutesialine — APPROVED** [review]

**Pull request merged**